webapp-client/src/component/Navbar.js

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from 'react';
+import '../stylesheets/Navbar.css';
+import Category from './Category';
+
+export default class Navbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.default = 'All';
+    this.state = {
+      categories: [this.default],
+      display: this.default,
+    };
+    this.addCategory = this.addCategory.bind(this);
+    this.setContent = this.setContent.bind(this);
+  }
+
+  addCategory(category) {
+    this.setState({
+      categories: [
+        ...this.state.categories,
+        category,
+      ]
+    });
+  }
```

Comment:
This, along with line 40- instead of passing in this function as a prop to AddCategoryForm, maybe move this function to addCategoryForm, and instead of having it update the local state or do anything, assume that this function will add to the actual DB, could be a TODO? so this functionality can be disconnected for now?  (This is if the restructuring we're talking about comes in the next PR)

Reply:
Yup I left a TODO and an explanation in the latest push:)